AudioAnalytics/AudioAnalyzer/src/main/java/sample/AudioAnalyzer.java

```diff
@@ -5,25 +5,22 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.ArrayList;
-import java.util.EnumSet;
 import java.util.List;
 import java.util.NoSuchElementException;
 import java.util.Scanner;
 import java.util.UUID;
-import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.Callable;
 import java.util.Arrays;
 
+import com.azure.storage.blob.*;
```

Comment:
We don't use star imports. I'd update your settings not to convert them to it.

Reply:
According to your comments, it has been fixed in the new version.